test/pages/Inventory.page.js

```diff
@@ -3,12 +3,49 @@ const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
+    // eslint-disable-next-line class-methods-use-this
```

Comment:
We should set lint config properly instead of adding tonns of such comments. Here you could read: https://eslint.org/docs/latest/rules/class-methods-use-this. Eslint tells us to use static methods since we have no `this` links in methods. You could read here (https://www.typescripttutorial.net/typescript-tutorial/typescript-static-methods-and-properties/) how to use it. Or remove this rule and keep going without these comments `// eslint-disable-next-line`